Applications/VpView/vpViewCore.cxx

```diff
@@ -814,34 +816,43 @@ void vpViewCore::exportTracksToFile()
     return;
     }
 
-  QString filter = "*.kw18;;*.json";
-  vgFileDialog fileDialog(0, tr("Export Tracks"), QString(), filter);
+  const auto session = this->SessionView->GetCurrentSession();
+  const auto* project = this->Projects[static_cast<size_t>(session)];
+  const auto* trackIO = project->ModelIO->GetTrackIO();
+  const auto* fseIO = project->ModelIO->GetFseTrackIO();
+
+  auto supportedTypes = trackIO->GetSupportedFormats();
+  if (fseIO)
+    {
+    supportedTypes.append(fseIO->GetSupportedFormats());
+    }
+  supportedTypes.append("All files (*)");
+
+  vgFileDialog fileDialog{qApp->activeWindow(), "Export Tracks",
+                          QString{}, supportedTypes.join(";;")};
   fileDialog.setObjectName("ExportTracks");
   fileDialog.setAcceptMode(QFileDialog::AcceptSave);
-  fileDialog.setDefaultSuffix("kw18");
+  fileDialog.setDefaultSuffix(trackIO->GetDefaultFormat());
   if (fileDialog.exec() != QDialog::Accepted)
     {
     return;
     }
 
-  QStringList files = fileDialog.selectedFiles();
+  const auto& filename = fileDialog.selectedFiles().first();
```

Comment:
BTW, this is a bug (dangling reference)... since it is introduced in this branch, remind me (if I forget) to also cherry-pick the fix into this PR after fixing the merge conflicts.

Reply:
```suggestion
  const auto filename = fileDialog.selectedFiles().first();
```

Reply:
Fixed in 89589a8.